src/null-safety/migration-guide.md

```diff
@@ -328,6 +329,9 @@ except for a 2.9 [version comment][].
 For more information about incremental migration, see
 [Unsound null safety][].
 
+Note that only fully migrated apps and packages 
+are compatible with Dart 3.
```

Comment:
Can we give them instructions on how to verify that they are fully migrated (without upgrading and then potentially downgrading again)?  e.g. "run your app and look for the 'running with sound null safety' message"?

Reply:
Good idea. Added under Step 5. below.